packages/blog-cms/package.json

```diff
@@ -21,7 +21,11 @@
     "@strapi/strapi": "^4.5.6",
     "lodash.set": "^4.3.2",
     "pg": "^8.8.0",
-    "strapi-provider-upload-azure-storage": "^2.1.0"
+    "react": "^18.2.0",
```

Comment:
were we missing dependencies?

Reply:
Yes, it's strapi that added these when trying to build